components/ReviewCard.tsx

```diff
@@ -74,16 +76,38 @@ export const DiffView: React.FC<DiffViewProps> = ({ diff = '' }) => {
 }
 
 const MemoDiffView = memo(DiffView)
-
+type UserProfileProps = {
+  username: string | undefined | null
+  name: string | undefined | null
+}
+const UserProfile: React.FC<UserProfileProps> = ({ username, name }) => {
+  //TO-DO fix User type to make these fields non-nulled
+  const firstName = name ? name.split(' ')[0] : ''
+  const lastName = name ? name.split(' ')[1] : ''
+  return (
+    <a
+      className={`${styles['comment_author']} mt-2 d-block`}
+      href={`/profile/${username}`}
+    >
+      <div className="d-inline">{`${firstName} ${
+        lastName ? lastName : ''
+      }`}</div>
+      <div className="d-inline text-muted">
+        {username ? '@' + username : 'Anonymous user'}
+      </div>
```

Comment:
Why the check on `lastName`? You already default it to an empty string.

Also put a space before the username otherwise it renders right next to the name.

```suggestion
      <div className="d-inline">{`${firstName} ${lastName}`}</div>
      {' '}
      <div className="d-inline text-muted">
        {username ? '@' + username : 'Anonymous user'}
      </div>
```

Reply:
I need the second check to `lastName`, otherwise it can end up undefined.
` const lastName = name ? name.split(' ')[1] : ''` defaults it to empty string is there is no `name`.
```lastName ? `${lastName} ` : ''``` defaults it to empty string if `name.split(' ')[1]` is undefined

Reply:
You can simplify the default assignement then.

`const lastName = name?.split(' ')[1] || ''`

or 

`const lastName = name?.split(' ')[1] ?? ''`

Reply:
Done.